geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/api/CoreOnlyUsesMembershipAPIArchUnitTest.java

```diff
@@ -105,12 +105,10 @@ public void miscCoreClassesDoNotUseMembershipInternals() {
 
   private void checkMembershipAPIUse(JavaClasses importedClasses) {
     ArchRule myRule = layeredArchitecture()
-        .layer("adapter")
-        .definedBy(resideInAPackage("org.apache.geode.distributed.internal.membership.adapter"))
         .layer("internal")
         .definedBy(resideInAPackage("org.apache.geode.distributed.internal.membership.gms.."))
         .layer("api").definedBy("org.apache.geode.distributed.internal.membership.api")
-        .whereLayer("internal").mayOnlyBeAccessedByLayers("api", "adapter");
```

Comment:
The "adapter" layer should _not_ be able to access "internal", so I removed the former.